### DISCUSSION TITLE
Does anyone know how to solve wglMakeCurrent() failed ?

### DISCUSSION THREAD

**Discussion by Nightinghost** [isl-org/Open3D#3863] [Q&A]
I use PyInstaller to package my gui-app code as an executable file (.exe) . It worked fine on my computer, but there was an error on the other computer.

![error_message](https://user-images.githubusercontent.com/58000426/128155509-069c8e39-9510-429c-a3b3-c9ca210fb1f9.jpg)

Does anyone know what's going on and how to fix it？
Thinks.

**Comment by gogo12345hhe** [comment]
whta's your gpu

**Reply by Nightinghost** [reply]
my gpu is Nvidia's, running correctly, but AMD has this problem

**Reply by gogo12345hhe** [reply]
Same problem

**Reply by Quanga** [reply]
**Build from source** solved it macos 10.15

**Comment by Quanga** [comment]
Same problem on AMD Radeon's and Intel.  Works on Nvidia.  Also looking for an assist.

**Comment by Jason190** [comment]
I got same problem. Is there any solution?
my CPU and GPU are both AMD's.
